bundles/binding/org.openhab.binding.temperlan/src/main/java/org/openhab/binding/temperlan/internal/TemperlanBinding.java

```diff
@@ -0,0 +1,228 @@
+				State temperature10Update = new DecimalType(state.getTemperature10());
+				State temperature11Update = new DecimalType(state.getTemperature11());
+				State temperature12Update = new DecimalType(state.getTemperature12());
+
+				// Send updates
+				if(state.getTemperature1() > -200) sendUpdate(configs, BindingType.temperature1, temperature1Update);
+				if(state.getTemperature2() > -200) sendUpdate(configs, BindingType.temperature2, temperature2Update);
+				if(state.getTemperature3() > -200) sendUpdate(configs, BindingType.temperature3, temperature3Update);
+				if(state.getTemperature4() > -200) sendUpdate(configs, BindingType.temperature4, temperature4Update);
+				if(state.getTemperature5() > -200) sendUpdate(configs, BindingType.temperature5, temperature5Update);
+				if(state.getTemperature6() > -200) sendUpdate(configs, BindingType.temperature6, temperature6Update);
+				if(state.getTemperature7() > -200) sendUpdate(configs, BindingType.temperature7, temperature7Update);
+				if(state.getTemperature8() > -200) sendUpdate(configs, BindingType.temperature8, temperature8Update);
+				if(state.getTemperature9() > -200) sendUpdate(configs, BindingType.temperature9, temperature9Update);
+				if(state.getTemperature10() > -200) sendUpdate(configs, BindingType.temperature10, temperature10Update);
+				if(state.getTemperature11() > -200) sendUpdate(configs, BindingType.temperature11, temperature11Update);
+				if(state.getTemperature12() > -200) sendUpdate(configs, BindingType.temperature12, temperature12Update);
+			
+		} catch (IOException e) {
+			logger.warn("Temperlan: cannot communicate with " + receiverProxy.getHost());
+		}
+	}
+
+	private Collection<TemperlanBindingConfig> getDeviceConfigs(
+			String deviceUid) {
+		Map<String, TemperlanBindingConfig> items = new HashMap<String, TemperlanBindingConfig>();
+		for (TemperlanBindingProvider provider : this.providers) {
+			provider.getDeviceConfigs(deviceUid, items);
+		}
+		return items.values();
+	}
+
+	private void sendUpdate(Collection<TemperlanBindingConfig> configs, BindingType type, State state) {
+		for (TemperlanBindingConfig config : configs) {
+			if (config.getBindingType() == type) {
+				eventPublisher.postUpdate(config.getItemName(), state);
+			}
+		}
+	}
+
+	@Override
+	protected void internalReceiveCommand(String itemName, Command command) {
+		
+		TemperlanBindingConfig config = getConfigForItemName(itemName);
+		if (config == null) {
+			logger.error("Received command for unknown item '" + itemName + "'");
+			return;
+		}
+		
+		TemperlanProxy proxy = proxies.get(config.getDeviceUid());
+		if (proxy == null) {
+			logger.error("Received command for unknown device uid '"
+					+ config.getDeviceUid() + "'");
+			return;
+		}
+
+		if (logger.isDebugEnabled()) {
+			logger.debug(BINDING_NAME + " processing command '" + command
+					+ "' of type '" + command.getClass().getSimpleName()
+					+ "' for item '" + itemName + "'");
+		}

```

Comment:
Use parameterized logger syntax here, which also eliminates the reason for `if (logger.isDebugEnabled())`.